internal/agent/config/config.go

```diff
@@ -148,3 +154,21 @@ func (cfg *Config) String() string {
 	}
 	return string(contents)
 }
+
+func (cfg *Config) PrepareStandalone() error {
```

Comment:
I think best would be if we can set default to:
`$HOME/.migration-planner/{data,config}` etc.
if it don't exist create it. If user set it's own path, ignore it.

Reply:
I think there shouldn't be any notion of `standalone` term anywhere :)